common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp

```diff
@@ -176,7 +190,13 @@ void AutowarePathFootprintDisplay::updateVehicleInfo()
   float width{property_vehicle_width_->getFloat()};
   float rear_overhang{property_rear_overhang_->getFloat()};
 
-  vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(length, width, rear_overhang);
+  if (vehicle_info_) {
+    vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(
+      vehicle_info_->vehicle_length_m, vehicle_info_->vehicle_width_m,
+      vehicle_info_->rear_overhang_m);
+  } else {
+    vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(length, width, rear_overhang);
```

Comment:
@takayuki5168 I'm sorry, but I should have minimized the scope. What do you think about it?

```suggestion
    const float length{property_vehicle_length_->getFloat()};
    const float width{property_vehicle_width_->getFloat()};
    const float rear_overhang{property_rear_overhang_->getFloat()};
    vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(length, width, rear_overhang);
```

Reply:
(I personally feel we can even delete these local variables.)

Reply:
I feel the same. I'll update.

Reply:
about minimizing the scope

Reply:
After that, I think you can merge this!

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/712/commits/c913b4610571e2f6ec46929dc9023a2c82cb26af
Done. Thank you so much!